_pages/time-table.html

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+permalink: /time-table/
+title: タイムテーブル
+---
+{% include navbar.html %}
+
+{% comment %} 
+  Jekyll プラグインで事前計算されたタイムテーブル表を使用
+  ロジックはプラグインで計算済み。Liquid は描画のみを担当
+{% endcomment %}
+
+{% assign tte         = site.data.time_table_events %}
+{% assign events      = tte.events %}
+{% assign rooms       = tte.rooms %}
+{% assign time_labels = tte.time_labels %}
+{% assign total_slots = tte.total_slots %}
+{% assign total_rooms = tte.total_rooms %}
+
+<section class="w-[calc(100dvw-40px)] max-w-7xl mx-auto mt-25 xl:mt-5">
+  <h2 class="text-4xl text-center mb-8">
+    Time table
+    <span class="block mt-3 text-2xl">タイムテーブル</span>
+  </h2>
+  <div class="overflow-x-auto border-[1px] border-[#e6e6e9]" aria-label="タイムテーブル（横スクロール可）">
+    <table class="ttable" style="--room-count: {{ rooms | size }};">
+      <caption>
+        {{ site.date_event }} のタイムテーブル
+      </caption>
+      <thead>
+        <tr class="sticky z-5">
```

Comment:
縦方向のスクロールがない(二重スクロールになっていない)ので、
ここのstickyは働きません。
ヘッダーの固定をしないならば削除で良さそうです
```suggestion
        <tr>
```